src/pointer/ptr.rs

```diff
@@ -1464,6 +1469,45 @@ mod _project {
     }
 }
 
+/// Implemented if `Ptr<T>` can be casted to `Ptr<U>` without violating the
+/// `UnsafeCell`-related invariants of [`Ptr`].
+///
+/// # Safety
+///
+/// `Ptr` requires that:
+///
+/// > During the lifetime 'a, no live reference or live `Ptr` will exist to this
```

Comment:
```suggestion
/// > During the lifetime `'a`, no live reference or live `Ptr` will exist to this
```

Reply:
Done (this particular doc comment no longer exists).